test/e2e/local-cluster/create_cluster.go

```diff
@@ -0,0 +1,221 @@
+	kubeconfigInternalName string
+
+	dockerImageRepository string
+	dockerImageTag        string
+	dockerImagePlatform   string
+
+	dockerImageBuildSkip = false
+
+	greenhouseControllerManagerNamespace string
+	greenhouseControllerManagerRelease   string
+
+	greenhouseControllerManagerValuesFilename string
+
+	idProxyNamespace      string
+	idProxyRelease        string
+	idProxyValuesFilename string
+)
+
+const (
+	TEST_TIMEOUT        = 3 * time.Minute
+	TEST_RETRY_INTERVAL = 3 * time.Second
+)
+
+func init() {
+
+	l := log.FromContext(context.Background())
+
+	flag.StringVar(&kindClusterName, "kindClusterName", "greenhouse-e2e", "Cluster name for creating a new kind cluster")
+
+	flag.StringVar(&kubeconfigName, "kubeconfigName", "e2e.kubeconfig", "kubeconfig file name for connecting to the e2e clusters")
+	flag.StringVar(&kubeconfigInternalName, "kubeconfigInternalName", "e2e.internal.kubeconfig", "kubeconfig file name for connecting to the e2e clusters from the same Docker network")
+
+	flag.StringVar(&dockerImageRepository, "dockerImageRepository", "greenhouse", "Docker image repository  for Greenhouse manager")
+	flag.StringVar(&dockerImageTag, "dockerImageTag", "e2e-latest", "Docker image tag for Greenhouse manager")
+	flag.StringVar(&dockerImagePlatform, "dockerImagePlatform", "linux/amd64", "Docker image platform for Greenhouse manager")
+	flag.BoolVar(&dockerImageBuildSkip, "dockerImageBuildSkip", false, "Skip building the docker image for Greenhouse manager")
+
+	flag.StringVar(&greenhouseControllerManagerNamespace, "greenhouseControllerManagerNamespace", "greenhouse", "Namespace for deploying Greenhouse manager")
+	flag.StringVar(&greenhouseControllerManagerRelease, "greenhouseControllerManagerRelease", "greenhouse", "Helm release name for deploying Greenhouse manager")
+
+	flag.StringVar(&greenhouseControllerManagerValuesFilename, "greenhouseControllerManagerValuesFile", "./manager.values.yaml", "path to the values file for greenhouse controller manager")
+
+	flag.StringVar(&idProxyNamespace, "gidProxyNamespace", "greenhouse", "Namespace for deploying idproxy")
+	flag.StringVar(&idProxyRelease, "idProxyRelease", "idproxy", "Helm release name for deploying idproxy")
+	flag.StringVar(&idProxyValuesFilename, "idProxyValuesFilename", "./idproxy.values.yaml", "path to the values file for idproxy")
+
+	flag.Parse()
+
+	l.Info("configuration loaded", "kindClusterName", kindClusterName, "dockerImageRepository", dockerImageRepository, "dockerImageTag", dockerImageTag, "dockerImageBuildSkip", dockerImageBuildSkip, "kubeconfigName", kubeconfigName, "kubeconfigInternalName", kubeconfigInternalName)
+
+}
+
+func main() {
+
+	ctx := context.Background()
+	l := log.FromContext(ctx)
+
+	// Create cluster
+	cluster := kind.NewCluster(kindClusterName)
+	cluster.SetDefaults()
+	kubeconfig, err := cluster.Create(ctx)
+	if err != nil {
+		l.Error(err, "cluster creation")
+		os.Exit(1)
+	}
+	l.Info("cluster created successfully")
```

Comment:
Can we also have logging for starting one of the steps.
E.g. "Starting cluster creation"

some steps take long and looking at the logs we do not really know what is happening atm.

Maybe have a look at[ how the `dev-env` spin-up logs](https://github.com/cloudoperators/greenhouse/blob/main/cmd/dev-env/main.go).

Is it possible to include the logs from `kind` or `docker` to get more verbose?